django_multitenant/mixins.py

```diff
@@ -217,3 +217,129 @@ def tenant_value(self):
     @property
     def tenant_object(self):
         return get_object_tenant(self)
+
+
+class DatabaseSchemaEditorMixin:
```

Comment:
Why did you need to move the classes in the base to here?

Reply:
See https://github.com/citusdata/django-multitenant/pull/150/#discussion_r1149440196 this is for reuse as these mixins provide the same functionality overrides for the Postgres and Postgis databases.